src/tests/dbus-tests/run_tests.py

```diff
@@ -67,6 +66,18 @@ def install_new_policy(projdir, tmpdir):
 
     return restore_list
 
+def install_new_dbus_conf(projdir, tmpdir):
+    '''Copies the DBus config file(s) to the system directory and backs up eventually the existing files.
+       Returns a list of files that need to be restored.'''
+    restore_list = []
+    config = os.path.join(projdir, "data/org.freedesktop.UDisks2.conf")
+    tgt = os.path.join('/etc/dbus-1/system.d/', os.path.basename(config))
+    if os.path.exists(tgt):
+        shutil.move(tgt, tmpdir.name)
+        restore_list.append(tgt)
+    shutil.copy(config, '/etc/dbus-1/system.d/')
+
+    return restore_list
 
 def restore_policy(restore_list, tmpdir):
```

Comment:
Nitpick: I would rename the function since it's not restoring the pollicy (only) now.